docs/content.zh/docs/learn-flink/streaming_analytics.md

```diff
@@ -180,7 +180,7 @@ Flink 有一些内置的窗口分配器，如下所示：
 以下都是一些可以使用的间隔时间 `Duration.ofMillis(n)`, `Duration.ofSeconds(n)`, `Duration.ofMinutes(n)`,
  `Duration.ofHours(n)`, 和 `Duration.ofDays(n)`。
 
-基于时间的窗口分配器（包括会话时间）既可以处理 `事件时间`，也可以处理 `处理时间`。这两种基于时间的处理没有哪一个更好，我们必须折衷。使用 `处理时间`，我们必须接受以下限制：
+基于时间的窗口分配器（包括会话时间）既可以处理 `事件时间`，也可以处理 `处理时间`。这两种基于时间的处理没有哪一个更好，我们必须折中。使用 `处理时间`，我们必须接受以下限制：
```

Comment:
google translate to english does not show any changes, but I see one character has changed. I assume the replacement is better Chinese grammatically.

Reply:
yes, the replacement is more widely used and more general